frontend/src/components/Interior/Conversations/NewConversation/TimeZones.js

```diff
@@ -0,0 +1,93 @@
+//Alex Cassell
+//http://alexcassell.com
+//set timezone for conversation
+import React from 'react';
+
+//grabs users timezone -- until default is set in preferences
+let getTimeZone = new Date().toString().match(/([A-Z]+[\+-][0-9]+.*)/)[1];
+/* converts getTimeZone to something like GMT-0400; Chrome likes to add a seemingly 
+random place within the timezone with the rest of the info */
+let timeZone = getTimeZone.substring(0,8);
+
+class TimeZones extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            timeZone : timeZone
+        };
+    }
+
+    handleTimeZone(e){
+        timeZone = e.target.name;
+        // conversation["timeZone"] = timeZone;
+        // console.log(conversation);
+        this.setState({timeZone:timeZone});
+    }
+
+    render() {
+        return (
+            /*eslint-disable */
+            <div className="conversations__timeZoneDropdown">
+            <button className="conversations__dropButton">Time Zone</button>
+            <div className="conversations__dropdown-content">
+                <div className="conversations__timezones__row">
+                    <a className="conversations__timeZone__square" name="LINT: UTC+14" onClick={(e) => this.handleTimeZone(e)} href="#">LINT: UTC+14</a>
```

Comment:
Same thing here. Pass in a list and print these in a loop. That would be really slick